cada/search.py

```diff
@@ -68,7 +68,6 @@
 }
 
 FIELDS = (
-    'id^5',
```

Comment:
Why this? From what I understand, people will not be able to search by case number anymore.

Reply:
because previously it was not in the search index. It's back.